geometries/intersection.json

```diff
@@ -0,0 +1,7 @@
+{
+	"id": 1108962113,
+	"name": "intersection",
```

Comment:
Is `intersection` a placetype or a geometry lat/lng or an integer `wof:id` for a specific intersection WOF feature?

Reply:
Can you add to the `description` property what you said in the comment above, please?

Reply:
done in most recent commit: https://github.com/whosonfirst/whosonfirst-geometries/pull/1/commits/3c660c7a3801b2809b3bb6c175362fa5220c3473.